app/forms/batch_edit_form.rb

```diff
@@ -57,6 +57,9 @@ def self.model_attributes(params)
     params[:rights] = Array(params[:rights]) if params[:rights].present?
     clean_params = super #hydra-editor/app/forms/hydra_editor/form.rb:54
     clean_params = encode_external_id(params, clean_params)
+    # These are removed due to a bug released in sufia 7.3:
+    # https://github.com/projecthydra-labs/hyrax/issues/652
+    clean_params['permissions_attributes'] = params['permissions_attributes'] if params['permissions_attributes']
     clean_params
```

Comment:
The word "sanitize" still makes me suspect this was meant to apply permissions controls; but I could be wrong and it never was, or the intent/design could be confused by multiple developers intending different things with it. 

But if this seems the best way to do it, that seems fine!  I _think_ this entire controller action should be protected at least to logged-in users, and all logged in users at present are staff, who can be trusted not to try to hack the system, so it's definitely fine at least for now. I'm good.  Working with sufia definitely involves lots of "good enough for now".